Automattic/wp-calypso
# Implement basic transfer step in new woo installer

#### Changes proposed in this Pull Request

* Implements a basic atomic transfer step in the new woo installer.

#### Testing instructions

You need a test site with a business plan on simple infra to begin this flow.

- http://calypso.localhost:3000/start/woocommerce-install/confirm?siteSlug=wooptestupgrade.wordpr\ess.com (change the site slug to your test site)
- Confirm the transfer ignoring the url change warning
- Transfer should eventually complete
- You should be redirected to wc-admin which automatically loads woo's onboarding "profile setup wizard"
- Your site should now be atomic with woo installed

To retest, use the revert tool in mc via blogrc, after the revert is triggered, it's good to refresh the page and click on the "Clean up Atomic Site Immediately" button to prevent any timing issues where you next test might get cleared by async job mid transfer.

Before

https://user-images.githubusercontent.com/811776/141722797-fb96270d-b2b5-435f-9d53-2078e5803c4b.mp4



After


https://user-images.githubusercontent.com/811776/141722826-4bf6a1d0-bb14-4b9c-b4f6-a91dce486183.mp4
